python/lsst/ts/cRIOpy/VMS/CSCPSDWidget.py

```diff
@@ -0,0 +1,178 @@
+# This file is part of cRIO/VMS GUI.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org). See the COPYRIGHT file at the top - level directory
+# of this distribution for details of code ownership.
+#
+# This program is free software : you can redistribute it and / or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CSCPSDWidget"]
+
+import math
+
+import numpy as np
+from PySide2.QtCore import Qt, Slot, QPointF
+from PySide2.QtCharts import QtCharts
+
+from ..GUI import AbstractChart
+from .ChartView import ChartView
+from ..GUI.CustomLabels import DockWindow
+from .Unit import units, coefficients
+
+
+class CSCPSDWidget(DockWindow):
+    """Display CSC calculated PSD.
+
+    Parameters
+    ----------
+    title : `str`
+        QDockWidget title and object name.
+    toolBar : `ToolBar`
+        Provides getFrequencyRange() method.
+    psd : `QSignal`
+        PSD SAL signal.
+    num_sensor : `int`
+        Number of sensors.
+    channles : `[int]`, optional
+        Channels to plot. Defaults to empty array, user should select channels
+        from context menu.
```

Comment:
I think it might not be a good idea to use mutable list as the default value:
https://docs.python-guide.org/writing/gotchas/

Reply:
Fixed there & anywhere else. Thanks.